packages/mgt/README.md

```diff
@@ -46,13 +46,13 @@ You can use the components by referencing the loader directly (via unpkg), or in
 ### Use via mgt-loader:
 
 ```html
-<script src="https://unpkg.com/@microsoft/mgt/dist/bundle/mgt-loader.js"></script>
+<script src="https://unpkg.com/@microsoft/mgt@2/dist/bundle/mgt-loader.js"></script>
 ```
 
 You can then start using the components in your html page. Here is a full working example with the Msal provider:
 
 ```html
-<script src="https://unpkg.com/@microsoft/mgt/dist/bundle/mgt-loader.js"></script>
+<script src="https://unpkg.com/@microsoft/mgt@2/dist/bundle/mgt-loader.js"></script>
```

Comment:
@gavinbarron do we need to add a note on what is defaulted to if one does not pin a version?

Reply:
I think this is a valid point.

Reply:
Great call out, thanks @musale.

I've updated both the root and mgt package readme files to include and expanded note on what the @2 fragment does and why it's needed.